content/2_Bronze/Complete_Rec.mdx

```diff
@@ -1015,10 +1029,10 @@ void dfs(int r) { // place queen in r-th row
 		++ans; // found valid placement
 		return;
 	}
-	for (int i = 0; i < 8; ++i) if (g[r][i] == '.' && !c[i] && !sum[i+r] && !dif[i-r+7]) {
-		c[i] = sum[i+r] = dif[i-r+7] = 1;
-		dfs(r+1);
-		c[i] = sum[i+r] = dif[i-r+7] = 0;
+	for (int i = 0; i < 8; ++i) if (g[r][i] == '.' && !c[i] && !sum[i + r] && !dif[i - r + 7]) {
```

Comment:
can you put the if statement in its own little block with curly braces and everything